server/database/dataGenerator.js

```diff
@@ -1,88 +1,99 @@
 const db = require('./index');
 
-/*-------- generates 100 records of dummy calendar availability data--------*/
+//clear db
+db.remove({}, (err) => {
+  if (err) {
+    throw err;
+  }
+});
 
-/* Example:
+//generate a random number between 0 and a given number
+const random = function(num) {
+  return Math.ceil(Math.random() * num);
+};
 
-Each document represents a year of availability data, each sub array of the year property represents a month
+//generate a price ($100 - 500)
+const price = function() { 
+  return 500 - random(400);
+};
 
-year: [
-  [jan],  -->  [1, 2, 3, 7, 9, 10, 16, 12]
-  [feb],        each month generates a random number of booked dates...
-  [mar],        ...based on the number of days in that month
-  [apr],  
-  [may],
-  [june],
-  [july],
-  [august],
-  [sept],
-  [oct],
-  [nov],
-  [dec]
-]
-*/
+//generate a cleaning fee [50, 75, 100, 125, 150]
+const cleaning = function() {
+  let options = [50, 75, 100, 125, 150];
+  return options[random(4)];
+};
 
-//generate a random number between 1 and a given number
-const randomNumberGenerator = function(num) {
-  return Math.ceil(Math.random() * num);
-}
+//generate a service fee (0% - 20% of price)
+const serviceFee = function(price) {
+  let percent = random(20) / 100;
+  return Math.ceil(price * percent);
+};
 
-//generates a single month of availability based on that month's length
-const generateDates = function(monthLength) {
-  //generate a random number between 1 and the length of the month
-  let length = randomNumberGenerator(monthLength);
-  //declare storage object
-  let store = {};
+//generate a year of availability info based on the current date
+const bookingsGen = function() {
   //declare output array
-  let output = [];
-  //while output array length is less than determined number of dates to book for the month
-  while (output.length < length) {
-    //generate a random number (date)
-    let newNum = randomNumberGenerator(monthLength);
-    //check if number has been used (by checking storage object)
-    if (!store[newNum]) {
-      //push num to output array
-      output.push(newNum);
-      //store num in storage object
-      store[newNum] = true;
+  let bookings = [];
+  //store current date info
+  let startDate = new Date();
+  //declare counter to track the current month through each iteration
+  let monthCounter = startDate.getMonth();
```

Comment:
Maybe use 'currentMonth' and 'currentYear' instead of 'monthCounter' and 'year', to make it clearer what they represent and that they are tracking similar information?